### PR TITLE
478: Added colors and clickability to filter string

### LINKED ISSUE #478: **Feature** Add colors to the search string  and make it clickable
Emphasize current filtering using severity colors (see style.css), and enhance navigation by making filer options clickable (automatically open relevant filter on click).

### PR DESCRIPTION
**Pull request by LironRS**
Closes #478: **Feature** Add colors to the search string and make it clickable

### CODE REVIEW

File: static/css/style.css [hunk 1]
```diff
@@ -689,4 +689,29 @@ input[type='date']{
...
 input[type='checkbox']{
     display: none;
-}
\ No newline at end of file
+}
+
+#filter-string{
+    text-align: center;
+    vertical-align: middle;
+    padding: 9%;
```
Comment (danielhers) [inline static/css/style.css]: Perhaps reduce to ~5% padding?
screenshot-2

File: static/js/app.js [hunk 2]
```diff
@@ -954,13 +954,25 @@ $(function () {
...
                 }
 
-                $("#filter-string").text(
-                    "" + "מציג " + markerCount + " תאונות בין התאריכים " + moment(this.dateRanges[0]).format('LL')
-                    + " עד " + moment(this.dateRanges[1]).format('LL') + severityText + fatal + severe + light
-                    + accuracyText + accurate + approx
-                );
+                $("#filter-string").empty()
+                    .append("<span>מציג </span>")
+                    .append("<span><a onclick='showFilter(0)'>"+markerCount+"</a></span>")
```
Comment (danielhers) [inline static/js/app.js]: Please define constants for the tabs (0, 1, 2)

File: static/js/app.js [hunk 3]
```diff
@@ -954,13 +954,25 @@ $(function () {
...
+                    .append("<span><a onclick='showFilter(1)' style='color: #d81c32;'>" + fatal + "</a></span>")
+                    .append("<span><a onclick='showFilter(1)' style='color: #ff9f1c;'>" + severe + "</a></span>")
+                    .append("<span><a onclick='showFilter(1)' style='color: #ffd82b;'>" + light + "</a></span><br>")
+                    .append("<span>" + accuracyText + "</span>")
+                    .append("<span><a onclick='showFilter(1)'>" + accurate + "</a></span>")
+                    .append("<span><a onclick='showFilter(1)'>" + approx + "</a></span>")
+                ;
             } else {
-                $("#filter-string").text(" התקרב על מנת לקבל נתוני סינון ");
+                $("#filter-string").empty()
```
Comment (danielhers) [inline static/js/app.js]: Isn't this equivalent to calling `.html(...)`?

File: static/js/marker.js [hunk 4]
```diff
@@ -141,6 +141,7 @@ var MarkerView = Backbone.View.extend({
                     }
                 }
 
+                console.log(this.model.get("locationAccuracy"));
```
Comment (danielhers) [inline static/js/marker.js]: Debug leftover?

File: templates/index.html [hunk 5]
```diff
@@ -1024,6 +1023,24 @@ <h4 class="title"></h4>
         inGooglePlay: 'בGoogle play'
     });
 </script>
+
+<script>
+    // show filter panel on search string click
+    function showFilter(number){
+        switch (number){
+            case 0:
```
Comment (danielhers) [inline templates/index.html]: See comment above about constants.
Or -- just pass the id instead of a number. Wouldn't that be more readable and less code?

**Comment by LironRS** [comment]
@danielhers I'll make all necessary changes tonight and update the branch. I need to make a habit on cross-browser testing, because on my chrome (macbook pro 13 inch screen) it look fine for all filterations. 
screen shot 2015-11-05 at 9 59 46 am

**Comment by danielhers** [comment]
You could try https://www.browserling.com/ :)

**Comment by LironRS** [comment]
@danielhers it's ready for final code review :)

**Pull request merged**